torchvision/datasets/stl10.py

```diff
@@ -45,8 +46,17 @@ class STL10(VisionDataset):
     ]
     splits = ('train', 'train+unlabeled', 'unlabeled', 'test')
 
-    def __init__(self, root, split='train', folds=None, transform=None,
-                 target_transform=None, download=False):
+    labels: np.ndarray
```

Comment:
I would have expected that this type annotation would go to the `__init__` of the class, as `labels` is not a class attribute but is instead an instance attribute?